Makefile.Protobuf.mk

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2023 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+# Generate gogo, swagger, go-validators, gRPC-storage-plugin output.
+#
+# -I declares import folders, in order of importance. This is how proto resolves the protofile imports.
+# It will check for the protofile relative to each of thesefolders and use the first one it finds.
+#
+# --gogo_out generates GoGo Protobuf output with gRPC plugin enabled.
+# --govalidators_out generates Go validation files for our messages types, if specified.
+#
+# The lines starting with Mgoogle/... are proto import replacements,
+# which cause the generated file to import the specified packages
+# instead of the go_package's declared by the imported protof files.
+#
+
+DOCKER_PROTOBUF_VERSION=0.5.0
+DOCKER_PROTOBUF=jaegertracing/protobuf:$(DOCKER_PROTOBUF_VERSION)
```

Comment:
there are already vars defined in the main Makefile for the same concept, should be using them.